src/main/java/com/sampleweb/pages/ComputersAndAccessoriesPage.java

```diff
@@ -0,0 +1,70 @@
+package com.sampleweb.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.Select;
+
+public class ComputersAndAccessoriesPage extends HomePage {
+    public String title = "Computers & Accessories";
+    private String pageTitle = "//*[@id=\"fst-hybrid-dynamic-h1\"]/div/h1/b";
```

Comment:
Can you use single quotes instead of double quotes. This eliminates the need to escape. 

e.g. 
Before:
//*[@id=\"fst-hybrid-dynamic-h1\"]/div/h1/b

After: 
//*[@id='fst-hybrid-dynamic-h1']/div/h1/b